SIMA/src/main.cpp

```diff
@@ -17,11 +18,16 @@ const int echoPin = 12;
 // Hall sensor pins
 const int HallSensorPin(2);
 
+// Debugging pins
+const int TX = 9;
+const int RX = 10;
```

Comment:
**Awesome !** Seems good to me.
Maybe a slight detail would be to name the `TX` and `RX` pins on this file with more infos, like `TX_Debug`, `TX_BT` or similar, so there is no confusion if another module uses UART (naming inside the class is perfect, it's just for the `main.cpp`)